### ISSUE TITLE
Newest Products Block button colors inherit from "Link color" but should be specific to buttons

### ISSUE THREAD

**Issue by cintacks** [Automattic/wp-calypso#66171]
### Quick summary

Global button style colors should apply correctly - but text takes on button background color. 

### Steps to reproduce

Prerequisite: WooCommerce installed with at least one product added. 

1. Enable Blank Canvas or Zoologist (only themes tested)
2. Add `Newest Products` Block to Editor
3. Adjust global button colors
4. View on frontend - button text color is same as button color

### What you expected to happen

Button text color should be set independently of `Link color`. 

### What actually happened

Button links take on "Link color" set in global styles, which has unintended effects (button text color is the same as button color)

### Context

User report: 5445100-zd-woothemes

Confirmed on test site, using default theme styles: https://shottr.cc/s/7BdN/SCR-20220802-m2up.png

### Theme-specific issue?

Blank Canvas
Zoologist

(maybe more)

### Reproducibility

Consistent

### Severity

Some (< 50%)

### Available workarounds?

Yes, easy to implement

### Workaround details

Provided CSS snippet for user - colorscan be modified as needed:

```
.wp-block-button a {
    color: white;
    text-decoration: none;
}

:is(.wp-block-search__button, .wp-block-button__link):is(:hover, :active, :focus) {
	background:#666;
	color:white;
}
```

**Comment by dolgelukkig** [comment]
Found another issue 
36180140-hc
Theme: Meraki
Block used: Hand-picked products

Shared CSS as a workaround.

**Comment by renata-franco** [comment]
📌 SCRUBBING

- Tested on AT ✅ 

------

📌 FINDINGS/SCREENSHOTS/VIDEO

I am not sure if I was able to replicate this issue or not. I think there is a very specific set of conditions for this to happen. I couldn't replicate with Blank Canvas but had some success with Zoologist. 

First, here is the color scheme I have currently on the test site: 

![imagen](https://user-images.githubusercontent.com/81378719/183937486-e6385418-a73e-4001-ba93-0ab4c6036b62.png)

And this gave me black background buttons for the products, with white text. However, when I hover the mouse on top of this, I get a black outline, and the background of the button gets white. Here is a GIF of the behavior: 

![CleanShot 2022-08-10 at 16 58 24](https://user-images.githubusercontent.com/81378719/183937918-3c2632af-f7f5-4093-89bc-97c6cb426123.gif)


I also tested the CSS provided for the workaround and it works for the hover, and it looks like this: 

![CleanShot 2022-08-10 at 17 01 33](https://user-images.githubusercontent.com/81378719/183938300-a4b9e1bb-cad6-454c-88e0-6e26fa3ea802.gif)

Another important thing would be the fact that this happened with other Product Blocks, not only Newest Products, which leads me to think that this is a Global Styles and/or theme issue. 


------

📌 ACTIONS
- Await author feedback

-------

📌 Message to Author

I have a few requests, please: 

1. Is it possible for you to provide a video of the process for you to replicate the issue? 
2. Please show us the conditions on your end: what theme, how your color palette looks like. 
3. Finally, can you test if this happens with other Product Blocks?

**Comment by cintacks** [comment]
You know what - I'm unable to replicate this now! So I suspect you're right about it needing a *very* specific set of conditions. I've tried to replicate with multiple conditions but have come up empty. 

I'll make sure to provide more visual evidence if I come across it again - my bad for rushing this bug report!

**Comment by renata-franco** [comment]
Thanks, @kettultim. I will be closing this one then. And it can be re-opened later in case it's necessary.
